tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
+
+[testenv]
+setenv =
+    # This allows us to reference test_settings.py
+    PYTHONPATH = {toxinidir}
+
+deps =
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    -rtest_requirements.txt
+
+commands =
+    django-admin.py test edx_rest_framework_extensions --settings=test_settings --with-coverage --cover-package=edx_rest_framework_extensions
+    coverage report
```

Comment:
@nedbat @stvstnfrd please provide examples. I have tried a number of different configurations, and cannot get coverage to run without having first run the tests. We are very near the point of diminishing returns—my working more on this to save 0.5s of runtime is not worthwhile—so, I am leaning toward improving this later.